config/jobs/kubernetes-csi/csi-proxy/csi-proxy-config.yaml

```diff
@@ -0,0 +1,27 @@
+# generated by gen-jobs.sh, do not edit manually
+
+presubmits:
+  kubernetes-csi/csi-proxy:
+  - name: pull-kubernetes-csi-csi-proxy
+    always_run: true
+    decorate: true
+    skip_report: false
+    skip_branches: []
+    labels:
```

Comment:
It looks like in this job config you need something like 

```
annotations:
  testgrid-dashboards: <dashboard>
```

See Testgrid's [config doc](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md) for more information